pyproject.toml

```diff
@@ -91,7 +93,9 @@ langchain = [
     "langchain",
 ]
 local_models = [
+    "neural-compressor",
     "optimum",
+    "optimum-intel",
```

Comment:
do we need to import those pkgs here. since you are lazy import them

Reply:
It's not either or, you can have everything "local" downloaded with `-E local_models` and if you don't, you get a helpful message.